test/suite/run.py

```diff
@@ -121,7 +121,8 @@ def usage():
   -h      | --help               show this message\n\
           | --hook name[=arg]    set up hooks from hook_<name>.py, with optional arg\n\
   -j N    | --parallel N         run all tests in parallel using N processes\n\
-  -l      | --long               run the entire test suite\n\
+  -l      | --long               run nearly the entire test suite (excludes test_sweep05)\n\
```

Comment:
Though it's only test_sweep05 that's covered by the new `--extra-long` command option so far, there could be other new tests being added to this bucket going forward. I'd prefer the text to be more generic e.g. "except for tests that run extra-long time".

Reply:
Yup good point! I changed it to "tests tagged with extralongtest" since the current --extra-long test runs all the long tests as well, so this might be easier to understand what is not included. Addressed here: [4f26984](https://github.com/wiredtiger/wiredtiger/pull/8957/commits/4f26984d1a5f17103ad9c833d74e0ac6879053c4)